java/java/JavaParser.g4

```diff
@@ -377,6 +410,12 @@ blockStatement
 
 localVariableDeclaration
     : variableModifier* typeType variableDeclarators
+    | variableModifier* VAR identifier '=' expression
+    ;
+
+identifier
+    : IDENTIFIER
+    | VAR
```

Comment:
It looks like all newly added keywords should be added to `identifier` as well. The following code is valid (you can check it online at [repl.com](https://replit.com/@IvanKochurkin/JavaKeywordsAsIdentifiers#Main.java)):

```java
class Main {  
  public static void main(String args[]) { 
    var module = "module";
    var open = "open";
    var requires = "requires";
    var exports = "exports";
    var opens = "opens";
    var to = "to";
    var uses = "uses";
    var provides = "provides";
    var with = "with";
    var transitive = "transitive";
    var var = "var";
    System.out.println(module);
    System.out.println(open);
    System.out.println(requires);
    System.out.println(exports);
    System.out.println(opens);
    System.out.println(to);
    System.out.println(uses);
    System.out.println(provides);
    System.out.println(with);
    System.out.println(transitive);
    System.out.println(var);
  } 
}
```

```suggestion
identifier
    : IDENTIFIER
    | MODULE
    | OPEN
    | REQUIRES
    | EXPORTS
    | OPENS
    | TO
    | USES
    | PROVIDES
    | WITH
    | TRANSITIVE
    | VAR
```

Also, I suggest adding this test to `examples`.